### ISSUE TITLE
Endpoint not found running integration tests when there is a global prefix

### ISSUE THREAD

**Issue by jorge-ribeiro-evaltec** [dj-nitehawk/FastEndpoints#58]
First I'd like to thank you very much for this amazing library!

Now onto the problem. We have a global API prefix, like so:
```csharp
        app.UseFastEndpoints(c =>
        {
            c.RoutingOptions = o => o.Prefix = "api";
        }
```

Until last week we were using version 3.5.1 and the integration tests were running fine. Then we upgraded to version 3.9.1 and now the routes are not found.

From what I could gather the routing options on the TestURLCache are missing the global prefix - and the routing options are not set when the endpoints are instantiated during the call to `builder.Services.AddFastEndpoints();`. The options are only set during the call to `UseFastEndpoints(...)`, when the TestURLCache is already built.

I can work around this using reflection, but it would be nice to have this working again. Maybe it would be easier to accept the routing options on the service call?

**Comment by dj-nitehawk** [comment]
should be fixed with `v3.10.0-beta3`. pls give that a try and report back. thanks!

**Comment by jorge-ribeiro-evaltec** [comment]
Just tried it and it is working perfectly now. Thanks!
